Lib/pathlib.py

```diff
@@ -13,16 +13,10 @@
 from urllib.parse import quote_from_bytes as urlquote_from_bytes
 
 
-supports_symlinks = True
-if os.name == 'nt':
-    import nt
-    if sys.getwindowsversion()[:2] >= (6, 0):
-        from nt import _getfinalpathname
-    else:
-        supports_symlinks = False
-        _getfinalpathname = None
+if os.name == 'nt' and sys.getwindowsversion()[:2] >= (6, 0):
```

Comment:
Is the `getwindowsversion` check still necessary? I think `GetFinalPathNameByHandle` is now available on all versions of Windows we support. @zooba am I right?

Reply:
My take: looks to be available from Vista onwards, so safe to assume it exists from python 3.5 onwards (where XP support was dropped)